langchain/src/chat_models/baiduwenxin.ts

```diff
@@ -0,0 +1,425 @@
+import axiosMod, { AxiosResponse, AxiosStatic } from "axios";
+import { BaseChatModel, BaseChatModelParams } from "./base.js";
+import {
+  AIChatMessage,
+  BaseChatMessage,
+  ChatGeneration,
+  ChatResult,
+  MessageType,
+} from "../schema/index.js";
+import fetchAdapter from "../util/axios-fetch-adapter.js";
+import type { StreamingAxiosConfiguration } from "../util/axios-types.js";
+import { CallbackManagerForLLMRun } from "../callbacks/manager.js";
+import { BaseLanguageModelCallOptions } from "../base_language/index.js";
+import { getEnvironmentVariable, isNode } from "../util/env.js";
+
+const axios = (
```

Comment:
Unfortunately this breaks builds in non-Node environments - can you use the built-in`fetch` like this? 

https://github.com/hwchase17/langchainjs/blob/main/langchain/src/llms/ai21.ts#L153

Reply:
Otherwise we'll have to mark it as Node only

Reply:
Ok, I will do it

Reply:
Hi, do you have suggest method to handle Event-Stream besides axios?

Reply:
Migrate all axios to fetch. Includes the event-stream things.